ZAPD/ZTexture.cpp

```diff
@@ -163,10 +163,22 @@ void ZTexture::ParseXML(XMLElement* reader)
 	ZResource::ParseXML(reader);
 
 	if (reader->Attribute("Width") != nullptr)
+	{
 		width = atoi(reader->Attribute("Width"));
+	} 
+	else
+	{
+		throw std::runtime_error("Width == nullptr");
+	}
```

Comment:
Would be nice if this error could provide a bit more information, like the name of the problematic resource. See [this](https://github.com/zeldaret/ZAPD/blob/c4773301a17a5a9f9c964a3e7ba6dfbf0730a94b/ZAPD/ZAnimation.cpp#L318) for an example.
Checking for invalid values would be nice too, instead of just crashing if someone writes something like `Width="0xC00"`.